pkg/provision/storage/asyncStorage.go

```diff
@@ -246,7 +246,8 @@ func (*AsyncStorageProvisioner) getAsyncWorkspaceCount(api provision.ClusterAPI)
 		return 0, 0, err
 	}
 	for _, workspace := range workspaces.Items {
-		if workspace.Labels[constants.DevWorkspaceStorageTypeLabel] == constants.AsyncStorageClassType {
+		storageClass := workspace.Spec.Template.Attributes.GetString(constants.DevWorkspaceStorageTypeLabel, nil)
```

Comment:
```suggestion
		storageClass := workspace.Spec.Template.Attributes.GetString(constants.DevWorkspaceStorageTypeAtrr, nil)
```
its not label anymore

Reply:
sorry, changed it to be `DevWorkspaceStorageTypeAtrr`